src/_components/pillButton/style.ts

```diff
@@ -20,3 +20,15 @@ export const PillButton = styled(Button)`
 export const PillButtonLabel = styled(Typography)`
   margin-left: 8px;
 `;
+
+export const PillButtonCount = styled(Typography)`
+  margin-left: 8px;
+  font-size: 10px;
```

Comment:
fontWeight en fontSize kunnen via props doorgegeven worden, niet? 
En eventueel leen minWidth opzetten (het bolletje lijkt nu niet 100% op een cirkel nu in de gif)